config/settings.yml

```diff
@@ -261,3 +261,6 @@ users_deletable_by_admins:
   default: 0
 users_deletable_by_self:
   default: 0
+journal_aggregation_time:
+  default: 300
```

Comment:
Ah! Perfect!

So we already have the same idea about our defaults ^^

Thinking about it we should either:

* rename the setting to something clumsy like `journal_aggregation_time_minutes`
* include a conversion into settings between the "frontend" and the "backend"
    * that means: settings in seconds, but UI in minutes

That is all supported by the idea, that the default unit of unitless time-measure is obviously seconds...